src/stats.c

```diff
@@ -204,19 +234,38 @@ mutex_stats_emit(emitter_t *emitter, emitter_row_t *row,
 
 #define EMITTER_TYPE_uint32_t emitter_type_uint32
 #define EMITTER_TYPE_uint64_t emitter_type_uint64
-#define OP(counter, type, human)					\
-	col = &col_##type[k_##type];						\
-	++k_##type;							\
-	emitter_json_kv(emitter, #counter, EMITTER_TYPE_##type,		\
-	    (const void *)&col->bool_val);
+#define OP(counter, type, human, synthetic, base_counter)		\
+	if (!synthetic) {                    \
+		col = &col_##type[k_##type];                        \
+		++k_##type;                            \
+		emitter_json_kv(emitter, #counter, EMITTER_TYPE_##type,        \
```

Comment:
We don't want the derived counters to be included for JSON format (e.g. similarly the `util` counter is not included in JSON output` -- they are parsed by applications and can be easily derived). Let's simply ignore them for JSON.

Reply:
They are not included into JSON as the if() statement at line 238 disables it. I can rename "synthetic" to be "derived" if it makes the code more readable.